api/v1beta1/gcpcluster_types.go

```diff
@@ -64,6 +64,10 @@ type GCPClusterSpec struct {
 	// supplied then the credentials of the controller will be used.
 	// +optional
 	CredentialsRef *ObjectReference `json:"credentialsRef,omitempty"`
+
+	// LoadBalancerSpec contains configuration for one or more LoadBalancers.
```

Comment:
Is this for more than one? It appears to only reference a single Load Balancer.

Reply:
The intent is that `LoadBalancerSpec` will contain all LoadBalancer configuration. Additional fields will be added to this struct including an array to define the scheme (Internal/External) and type (passthrough/proxy) of multiple Load Balancers. This initial `APIServerInstanceGroupTagOverride` applies to the APIServer instance group across any load balancers